libraries/AP_Scripting/applets/arming-checks.md

```diff
@@ -0,0 +1,63 @@
+# arming_checks
+
+This script implements user defined pre-arm and arming checks. Checks
+can be warnings or errors. Warnings don't prevent arming but will display
+on the ground station, whereas errors will prevent arming.
+
+Additional arming checks can be added by adding a method that returns true/false (true = arming ok), and defining the check, including the parameter it uses in the method arming_checks().
+
+# Parameters
+
+There is a single parameter defined for each arming check. The error or warning level can be changed for each arming check by setting the MAV_SEVERITY of the arming check. An arming check can be disabled by setting the parameter value to MAV_SEVERITY.NONE or -1.
+
+All arming check parameters are prefixed with ZAR_
+Z = Scripting Parameter
+AR = Arming Checks
+
+# ZAR_SYSID
```

Comment:
I'm not a fan of the ZAR prefix, maybe ACHK_ ?

Reply:
I like it - since you aren't requesting a change ...